web/src/features/hfiCalculator/pages/HfiCalculatorPage.tsx

```diff
@@ -277,6 +278,7 @@ const HfiCalculatorPage: React.FunctionComponent = () => {
         </Container>
       ) : (
         <Container maxWidth={'xl'}>
+          <LastUpdatedHeader dailies={dailies} />
```

Comment:
@tessmgood having the last update date here (as opposed to on the day in question) could cause some confusion. For example - if I click on any of the prep days, the date displayed here could possible relate to a different prep day.